src/react/components/common/assetPreview/imageAsset.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { IAssetProps } from "./assetPreview";
+
+export interface IImageProps extends IAssetProps, React.ClassAttributes<ImageAsset> {
```

Comment:
Is this just to have for future use?

Reply:
It's in use, just doesn't have any of its own members... it implements both `IAssetProps` and regular react props.